config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml

```diff
@@ -50,3 +50,17 @@ presubmits:
         - "--scenario=execute"
         - "--"
         - "./scripts/ci-test.sh"
+  - name: pull-cluster-api-vendor-in-sync
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/cluster-api:v20181020-5ce477da2
```

Comment:
@roberthbailey Wo builds this image? The tag I used is only correct if it will be built today at commit 5ce477da2.

Does it work like this or do I have to do two prs, one for the image change, one for the presubmit?

Reply:
@BenTheElder can confirm but I think you can do it in one commit as long as the new image has already been pushed with the relevant changes inside of it.

Reply:
Also, you should update the image in the other tests so that we use the same image everywhere.

Reply:
> @BenTheElder can confirm but I think you can do it in one commit as long as the new image has already been pushed with the relevant changes inside of it

Yeah but who does push it? I see no job on the pull that does it.

I'll update the tags for the other images when I know what it should be.

Reply:
sorry for the delay, my inbox is RIP..

I have no idea who maintains this image.. looks like krousey?

https://github.com/kubernetes/test-infra/blob/be2f242ddabc373ecd0e156d456cb8def906dfd5/images/cluster-api/Dockerfile#L17

These are not auto pushed. I can push one though, and I'm pretty sure you have permissions @roberthbailey

Reply:
@roberthbailey Could you please push an image with my changes and tell me the tag, so I can update the PR and we can merge it?